replicated-security/stride-ics-testnet-two-1/readme.md

```diff
@@ -0,0 +1,113 @@
+
+```sh
+git clone https://github.com/Stride-Labs/stride.git
+cd stride
+git checkout a3eff2dc
+make install
+strided init stride-node --chain-id stride-ics-testnet-two-1
+
+# Grab the genesis file
+curl -L https://raw.githubusercontent.com/Stride-Labs/mainnet/ics-testnet/ics-testnet/genesis.json -o $HOME/.stride/config/genesis.json
+```
+
+* Start stride node, node should start catching up
+* Node will panic at UPGRADE_HEIGHT_TBD
+* Stop the node
+
+```sh
+cd $HOME/stride
+
+git checkout 5a45a4cfaef54aa0e875eb4e618bfae882d278a2
+
+make install
+```
+
+replace the binary
+
+```sh
+mkdir -p $HOME/.sovereign/config
+
+curl -L https://github.com/cosmos/testnets/raw/master/replicated-security/stride-ics-testnet-two-1/genesis.json -o $HOME/.sovereign/config/genesis.json
+```
+</details>
+
+<details><summary>Detailed steps for transitioning Stride node from non-validator on Stride testnet to validator on consumer chain</summary>
+<br>
+
+_Thanks to Bosco from Silk Nodes_
+
+Download v11 Binary
+```sh
+cd stride
+git pull
+git checkout 5a45a4cfaef54aa0e875eb4e618bfae882d278a2
+make install
+
+#Should be v11
+strided version
+```
+
+Make directories in cosmovisor and copy binaries
+```
+mkdir -p $HOME/.stride/cosmovisor/upgrades/v11/bin/
+cp $HOME/go/bin/strided $HOME/.stride/cosmovisor/upgrades/v11/bin/
+```
+
+Download new Sovereign genesis
+```
+mkdir -p $HOME/.sovereign/config/
+wget -O $HOME/.sovereign/config/genesis.json https://cdn.discordapp.com/attachments/1064857924402413600/1116022613966336041/genesis_new.json
+```
+
+Restart the Service
+```
+sudo service stride restart && journalctl -u stride -f -o cat
+```
+
+</details>
+
+# Launch Sequence
+
+| # | When? | Provider side | Stride side |
+| -- | --- | ----- | ---- |
+| 1 | Now | | Join the Stride testnet `stride-ics-testnet-two-1` with the pre-transition binary (commit hash `a3eff2dc`) as a full node (not validator) and sync to the tip of the chain (link to instructions below). |
+| 2 | Now until software upgrade proposal passes on Stride | | Build (or download) the target (post-transition) Stride binary. <br><br>If you are using Cosmovisor, place place it in Cosmovisor `/upgrades/<upgrade-name>/bin` directory.<br><br>If you are not using Cosmovisor, be ready to manually switch your binary at the upgrade halt height. |
+| 3 | Now until spawn time (15:00 UTC) | Submit assign-consensus-key for stride-ics-testnet-two-1 with the keys on your full node (**note: make sure to do this before spawn time!)** | |
+| 4 | During voting period for  consumer-addition proposal on provider | You don’t have to do anything. Optionally, you may vote for the consumer-addition proposal | |
+| 5 | During voting period for software upgrade on Stride | | You don’t have to do anything. |
+| 6 | After spawn time | | Place the newly generated “genesis” file (containing only the ccv state) in the ⚠️ `$NODE_HOME/config directory` ⚠️ Stride will provide this.<br><br>Do NOT replace the existing genesis file. |
```

Comment:
This is different from the "Download new Sovereign genesis" step from the above instructions- is it `$NODE_HOME/config` or `$HOME/.sovereign/config`?